docs/GOVERNANCE.md

```diff
@@ -0,0 +1,139 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the
+`napari` project, to clarify how decisions are made and how the various
+elements of our community interact.
+
+This is a consensus-based community project. Anyone with an interest in the
+project can join the community, contribute to the project design, and
+participate in the decision making process. This document describes how that
+participation takes place, how to find consensus, and how deadlocks are
+resolved.
+
+# Roles And Responsibilities
+
+## The Community
+The napari community consists of anyone using or working with the project
+in any way.
+
+## Contributors
+
+A community member can become a contributor by interacting directly with the
+project in concrete ways, such as:
+
+- proposing a change to the code via a GitHub
+  [GitHub pull request](https://github.com/napari/napari/pulls);
+- reporting issues on our
+  [GitHub issues page](https://github.com/napari/napari/issues);
+- proposing a change to the documentation, or
+  [tutorials](https://github.com/napari/napari-tutorials) via a
+  GitHub pull request;
+- discussing the design of the napari or its tutorials on in existing
+  [issues](https://github.com/napari/napari/issues) and
+  [pull requests](https://github.com/napari/napari/pulls);
+- discussing examples or use cases on the
+  [image.sc forum](https://forum.image.sc/tags/napari) under the #napari tag; or
+- reviewing [open pull requests](https://github.com/napari/napari/pulls)
```

Comment:
how about zulip?

Reply:
probably worth mentioning here, but i'd prefer to push user / examples discussions to `image.sc` and limit zulip to minimal dev coordination as much as possible as I've seen people almost burn out from user requests in a real time chat system. curious how others feel

Reply:
changed, with an added comment about preference for asynchronous communication